server/services/partnerComplianceGuard.ts

```
@@ -0,0 +1,199 @@
+          error: "Transfer blocked",
+          message: result.reason,
+          details: {
+            partnerName: result.partnerName,
+            complianceStatus: result.complianceStatus,
+            lockReason: result.lockReason,
+          },
+        });
+      }
+      next();
+    })
+    .catch(error => {
```

ESLint

[error] Replace `(error` with `((error)`

(prettier/prettier)